app/views/development_metrics/defect_escape_rate/_values_details.erb

```diff
@@ -16,10 +16,10 @@
       <hr>
       <div>
         <h5>Bugs by environment:</h5>
-        <% metric[:values].each do |metric| %>
+        <% metric[:values].each do |environment, bugs| %>
           <div class="item-list">
-            <%= metric.first.to_s.capitalize %>:
-              <%= metric.second.to_s %>
+            <%= environment.to_s.capitalize.presence || 'None' %>:
+            <%= bugs.second.to_s %>
```

Comment:
shouldn't this be
```suggestion
            <%= bugs %>
```

now that you changed `|metric|` for `|environment, bugs|` ?

Reply:
Haha yes 😅